turbinia/workers/plaso.py

```diff
@@ -36,6 +36,45 @@ class PlasoTask(TurbiniaTask):
       state.DECOMPRESSED
   ]
 
+  task_config = {
+      'status_view': 'none',
+      'hashers': 'all',
+      'partition': 'all',
+      'vss_stores': 'all',
+  }
+
+  def build_plaso_command(self, base_command, conf):
+    """Builds a typical plaso command, contains logic specific to log2timeline.
+
+    Args:
+      base_command (str): Command to invoke log2timeline (e.g. log2timeline.py)
+      conf (dict): Dynamic config containing the parameters for the command.
+
+    Returns:
+      String for valid Log2timeline command.
+    """
+    cmd = [base_command]
+    for k, v in conf.items():
```

Comment:
How is this affected by the proposal above of having recipe globals? (imo the only really fundamental open piece in this PR). It makes sense to leverage the patterns in command creation where consistency exists, and this could  then include interpreting recipe globals in whatever terms applicable to the tool at hand (or not).

By all means let's discuss how to handle recipe globals. Two options that come to mind are creating a virtual recipe item that applies, and is invoked by, all the pieces of code that invoke recipe/config items. This would be passed along with the working recipes dict with a dedicated key. Alternatively we could keep it in the global config, and see how much more demand for this we encounter?

Reply:
Agreed on the line list + skip logic for the command building routine.

I am not sure what came over me.. What a convoluted mess of a paragraph, haha. Yes, we are saying essentially the same. 

Giving a try to including a 'global' key that will contain globally applicable recipe items, and the sub-keys within that global recipe item would be merged with the working recipe for the task when passed. let me know what you think about the merging. The rationale is to keep the task code from accessing the _config dict passed along with the evidence.